language/tools/move-unit-test/tests/test_sources/timeout.exp

```diff
@@ -3,7 +3,7 @@ Running Move unit tests
 [ FAIL    ] 0x1::M::no_timeout_fail
 [ PASS    ] 0x1::M::no_timeout_while_loop
 [ TIMEOUT ] 0x1::M::timeout_fail
-[ TIMEOUT ] 0x1::M::timeout_fail_with_expected_failure
+[ PASS    ] 0x1::M::timeout_fail_with_expected_failure
```

Comment:
This doesn't seem right -- is it expected?

Reply:
This is the from the switch you suggested, about having expected failure catching everything, including out of gas